internal/search/crawler/github/crawler.go

```diff
@@ -0,0 +1,577 @@
+// Package github implements the crawler.Crawler interface, getting data its
+// from the Github search API.
+package github
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"math"
+	"net/http"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/damienr74/kustomize/internal/search/doc"
```

Comment:
Can you change it to `sigs.k8s.io/kustomize`? Then when we move it to the kustomize repo later, we don't need to update the import path.

Reply:
The reason why it was set up this way was to be able to actually compile/test all of this code locally. With the wrong local path, I had issues with gomodules, but I can try again and see if I manage to get it working.

Reply:
This was a problem from before I figured out how to implement multiple modules in a repo. Fixed, thanks for bringing it up!